libraries/gpu/src/gpu/Texture_ktx.cpp

```diff
@@ -88,8 +87,11 @@ struct GPUKTXPayload {
         memcpy(&_usageType, data, sizeof(TextureUsageType));
         data += sizeof(TextureUsageType);
 
-        memcpy(&_originalSize, data, sizeof(glm::ivec2));
-        data += sizeof(glm::ivec2);
+        if (version >= 2) {
+            memcpy(&_originalSize, data, sizeof(glm::ivec2));
+            data += sizeof(glm::ivec2);
+        }
```

Comment:
Is the object re-used? If so, does `_originalSize` need to be set to `0, 0` for `version < 2`?

Reply:
set default value above